### ISSUE TITLE
Problem with Geomagic & SofaPython3 plugin

### ISSUE THREAD

**Issue by Gronum00** [sofa-framework/sofa#3185]
Greetings,

	I am experiencing a problem with Geomagic plugin. Actually it does work fine on its own, but the problem begins when I add the SofaPython3 plugin. After that, the first 3 angle values (first 3 joints from the base) of the simulation model of geomagic haptic device does not work. However, last 3 passive joints works fine. And in neither case I get an error message.
	I was wondering if anyone else experience the same problem? Or is there a solution?

	Also is there any way to use the haptic device with a python code for Sofa. Because our aim is to use geomagic haptic devices as the masters for a soft robot that is simulated via SoftRobots plugin.

**Comment by epernod** [comment]
Hi,

Could you put your scene example here and the error message you get.
The Geomagic plugin just create SOFA components to be used in the scene graph so there should be no interference with the SofaPython3 mechanism.
But nothing is better than testing! 😄

**Comment by Gronum00** [comment]
Hey there,

Sorry for the late response. I have solved the problem. One of the forum posts (cannot find which one or even on which forum) mentioned about computer's "time zone" and  "language" and "formats" settings being effective on using Geomagic devices. 

Although they worked fine in their own demo software, and with Sofa Framework; the devices were not working with SofaPython3 plugin.

I have no idea why but the problem was with the "formats" settings of my computer. When I change it to "United States" the problem was resolved.
